doc/conf.py

```diff
@@ -321,7 +321,7 @@ def __getattr__(cls, name):
     'matplotlib', 'matplotlib.pylab', 'tifffile', 'EdfFile', 'netCDF4',
```

Comment:
Ah.. Can we simply put pass inside these functions and keep their name for some time?